smoketest.sh

```diff
@@ -206,11 +206,14 @@ runDemoApps() {
         --env QUARKUS_HTTP_PORT=10010 \
         --env ORG_ACME_CRYOSTATSERVICE_ENABLED="false" \
         --env CRYOSTAT_AGENT_APP_NAME="quarkus-test-agent-1" \
-        --env CRYOSTAT_AGENT_WEBCLIENT_SSL_TRUST_ALL="true" \
-        --env CRYOSTAT_AGENT_WEBCLIENT_SSL_VERIFY_HOSTNAME="false" \
+        --env CRYOSTAT_AGENT_WEBCLIENT_TLS_TRUST_ALL="true" \
+        --env CRYOSTAT_AGENT_WEBCLIENT_TLS_VERIFY_HOSTNAME="false" \
         --env CRYOSTAT_AGENT_WEBSERVER_HOST="localhost" \
         --env CRYOSTAT_AGENT_WEBSERVER_PORT="9977" \
-        --env CRYOSTAT_AGENT_CALLBACK="http://localhost:9977/" \
+        --env CRYOSTAT_AGENT_WEBSERVER_TLS_KEYSTORE_PASS="/certs/keystore.pass" \
+        --env CRYOSTAT_AGENT_WEBSERVER_TLS_KEYSTORE_FILE="/certs/crysotat-keystore.p12" \
+        --env CRYOSTAT_AGENT_WEBSERVER_TLS_CERT_FILE="/certs/server.cer" \
```

Comment:
If the quarkus-test application is going to have certs added to it like this, then the cert should either be embedded into the application build itself (not coming from the agent, but rather the combination of raw application plus agent into a container image), or the cert should be overlaid on top of the application container image by using a `--mount` like in `run.sh`. If you go the `--mount` way then the certificate used can actually be generated within the smoketest script, or the script can check if a cert exists in some local directory.